section2/reflection.md

```diff
@@ -1,15 +1,32 @@
+- SMART goal setting is something that has resonated with me over time, throughout most (if not all) of my academic settings, in order to limit the amount of overwhelm I feel within projects and to achieve them by specific deadlines by use of sub-goals.
+- Understanding that mistakes are essential to learning is a point that was brand new to me. Most often in my past academic settings, there is a level of judgement and near-embarrassment applied when presenting my work. Most often, my work was always presented as a "final submission" or that it should reach perfection by the time anyone else can view the project. It's such a new feeling to understand that not all of my coding knowledge or technique will be perfect, and is necessary to seek help to achieve efficiency and learning throughout the coding world.
 
-2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+### 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+- I currently demonstrate a Growth Mindset by beleiving that I will improve my coding skills with effort and practice. This is something that, despite challenges, is one of the most rewarding parts of learning to code in my opinion. This keeps me having a positive attitude and continually seeking to learn and improve what I can.
+- I also understand that all minds will take a different approach in some unique way when reviewing a project and how vital that is to the process of solving or completing said work. Besides, as mentioned, before mistakes are vital to learning. I am actually starting to enjoy making these mistakes purely by learning through Mod0 and collaborating. I have always loved team-based activities, and these mistakes provide ways for me to self-teach new skills on accident and to also learn how to learn with others.
+- I am also starting to learn how to seek out challenges, especially when working on this Mod0 Project. Seeking out challenges helps me to understand more and more of what does and does not make a certain code block work, fail, or alter in some way or another. I haven't always done things this way, but it's certainly an exciting new way to learn!
+- Something I am not demonstrating a Growth Mindset is fully not being afraid to seek help. I'm not exactly unwilling to seek help, but I am not fully comfortable with knowing how and when to ask questions. Sometimes, I feel that I can't yet articulate what my issue is that I am having or where in the process I am lost/made a mistake. This is something that has been highly beneficial while being in Mod0, as I'm able to "phone a friend" and verbally as well as visually display my code with others to be able to talk through the process and understand what exactly is the part of the instructions I may have overlooked, where there is a gap in understanding, or to help me be able to articulate what exactly I need and which resources to find in order to fix it.
```

Comment:
Yes, forming the question itself is an ENTIRE SKILL! It's hard. Many times I find that I can answer my own question once I do all the work of typing something coherent up in a Slack message or "rehearsing" how I'll ask it on a zoom, just because I had to do so much thinking in order to form the question/plan on how I'd share the context.